SofaImGui/src/SofaImGui/ImGuiGUIEngine.cpp

```diff
@@ -117,7 +117,7 @@ void ImGuiGUIEngine::init()
     ini.SetUnicode();
     if (sofa::helper::system::FileSystem::exists(sofaimgui::AppIniFile::getAppIniFile()))
     {
-        SI_Error rc = ini.LoadFile(sofaimgui::AppIniFile::getAppIniFile().c_str());
+        [[maybe_unused]] SI_Error rc = ini.LoadFile(sofaimgui::AppIniFile::getAppIniFile().c_str());
```

Comment:
I disagree on the fix. 

To remove the warning we need to replace the assert with a real error checking.